mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceInterfaceDTO.java

```diff
@@ -20,14 +20,18 @@
 package io.github.ust.mico.core.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-
+import io.github.ust.mico.core.configuration.extension.CustomOpenApiExtentionsPlugin;
+import io.github.ust.mico.core.model.MicoServiceInterface;
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.annotations.Extension;
+import io.swagger.annotations.ExtensionProperty;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
 
 /**
- * Represents a service in Kubernetes for a {@link io.github.ust.mico.core.model.MicoService}.
+ * DTO for status information of a {@link MicoServiceInterface}, that is mapped to a Kubernetes Service.
```

Comment:
I think I would make sense to add more information to this DTO like the IP addresses.

Reply:
I will add a field **publicIps** to MicoServiceInterfaceStatusDTO and implement it in MicoStatusService equivalent to the endpoint `getInterfacePublicIpByName`. We should think about extracting the logic of this endpoint to avoid duplicate code (part of #491 ?).

Reply:
ok. I added this information to the description of #491.